build.sbt

```diff
@@ -319,6 +320,38 @@ lazy val `kamon-cassandra` = (project in file("instrumentation/kamon-cassandra")
     )
   ).dependsOn(`kamon-core`, `kamon-instrumentation-common`, `kamon-testkit` % "test", `kamon-executors`)
 
+lazy val `kamon-elasticsearch` = (project in file("instrumentation/kamon-elasticsearch"))
+  .disablePlugins(AssemblyPlugin)
+  .enablePlugins(JavaAgent)
+  .settings(instrumentationSettings)
+  .settings(
+    fork in (Test,run) := true,
+    libraryDependencies ++= Seq(
+      kanelaAgent % "provided",
+      "org.elasticsearch.client" % "elasticsearch-rest-client" % "7.9.1" % "provided",
+      "org.elasticsearch.client" % "elasticsearch-rest-high-level-client" % "7.9.1" % "provided",
+      scalatest % "test",
+      logbackClassic % "test",
+      {
+        if(scalaBinaryVersion.value == "2.11") {
+          "com.dimafeng" % "testcontainers-scala_2.11" % "0.38.3" % "test"
```

Comment:
Just use %% `"com.dimafeng" %% "testcontainers-scala"`

Reply:
I swear I tried that and it didn't work... I must have forgotten for use `sbt reload` or something. 

Thx for the feedback :+1: